x86_symbolic/src/Data/Macaw/X86/Crucible.hs

```diff
@@ -869,6 +872,11 @@ unpack2 sym e w c v1 v2 k =
        v2' <- getBitVal sym v2
        k (V.fromBV e n c v1') (V.fromBV e n c v2')
 
+-- | Extract the offset from an LLVMPointer
+--
+-- Note that this version generates an /assertion/ that the block number is
+-- zero, and should be used carefully.
```

Comment:
It would be worth mentioning `getPointerOffset` here as an alternative that doesn't generate an assertion.